### DISCUSSION TITLE
Editeur de fichier XML kodi

### DISCUSSION THREAD

**Discussion by detobel36** [Kodi-vStream/venom-xbmc-addons#3934] [Support & aide]
Bonjour,

Je voulais savoir s'il n'existait pas un editeur de fichier XML pour Kodi ?

Typiquement pour "prévisualiser" un fichier comme celui-ci: https://github.com/Kodi-vStream/venom-xbmc-addons/blob/Beta/plugin.video.vstream/resources/settings.xml

**Comment by sizanic** [comment]
Je n'en connais pas dans Kodi.
Mais si je devais éditer un fchier xml, je le ferais à l'exterieur de kodi avec un éditeur dédié, rien que pour la gestion du clavier qui ne serait que meilleure.

**Comment by detobel36** [comment]
Pardon, je me suis mal exprimé.

Je cherche un "prévisualisateur" pour développer.  Un editeur à installé (sur linux/windows) qui permet de voir "en avance" ce que va rendre le XML dans kodi.

**Comment by sizanic** [comment]
J'ai regardé parmi cette liste classée par sous-catégorie, je n'ai rien trouvé : 

https://kodi.wiki/view/Category:All_add-ons

**Comment by TmpName** [comment]
Et un truc en ligne ? https://codebeautify.org/xmlviewer

**Reply by detobel36** [reply]
> Et un truc en ligne ? https://codebeautify.org/xmlviewer

Ca ne permet pas du tout d'avoir une idée de comment "ça va rendre" dans Kodi :stuck_out_tongue_closed_eyes:
